pkg/limayaml/defaults_test.go

```diff
@@ -211,27 +212,27 @@ func TestFillDefault(t *testing.T) {
 	expect.Mounts[0].NineP.ProtocolVersion = ptr.Of(Default9pProtocolVersion)
 	expect.Mounts[0].NineP.Msize = ptr.Of(Default9pMsize)
 	expect.Mounts[0].NineP.Cache = ptr.Of(Default9pCacheForRO)
-	expect.Mounts[0].Virtiofs.QueueSize = ptr.Of(DefaultVirtiofsQueueSize)
+	expect.Mounts[0].Virtiofs.QueueSize = nil // Setting QueueSize to default requires vmType == QEMU and MountType == virtiofs
```

Comment:
I don't think the PR needs to be split, but it might be a good idea to split the commits.

Reply:
I split the commits.

Reply:
We usually want commits to be squashed, so moving unrelated changes to a separate PR makes sense, even if the changes are very small.

Reply:
FWIW, I'm fine with keeping a couple very small fixes unsquashed in a single PR as long as @AkihiroSuda approves...

Reply:
> Even in cases where the test would fail without that change?

No, because that would prevent CI from passing. As I wrote above, I'm flexible with this rule; it is @AkihiroSuda who introduced it.

Reply:
Squashing before merging is fine because the comments added when splitting the commits will remain intact. 👍🏻